.github/workflows/cicd.yml

```diff
@@ -37,3 +37,36 @@ jobs:
             build/tmp/deploy/sdk/HULKs-OS-x86_64-toolchain-${{ github.ref_name }}.sh
             build/tmp/deploy/sdk/HULKs-OS-aarch64-toolchain-${{ github.ref_name }}.sh
           fail_on_unmatched_files: true
+  devimage:
+    name: Build and release development image
+    if: startsWith(github.ref, 'refs/tags/')
```

Comment:
use this instead:

```
on:
  push:
    tags:
      - v[0-9]+.[0-9]+.[0-9]+   
```

Reply:
Are you sure this is correct (when removing the v)? The ´on´ key seems to exist only for the entire workflow resulting in it only running for releases.

Reply:
I am not sure. Is there nothing similar for only jobs?!

Reply:
It is not listed [here](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions), so probably not.